embedded-graphics/CHANGELOG.md

```diff
@@ -32,6 +32,11 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 - [#317](https://github.com/jamwaffles/embedded-graphics/pull/317) The bounding box size for `Circle`s was off by one.
 
+### Removed
+
+- **(breaking)** [#351](https://github.com/jamwaffles/embedded-graphics/pull/351) Removed the `egtext!` and `text_style!` macros. Use the `Text` and `TextStyleBuilder` respectively instead.
+- **(breaking)** [#351](https://github.com/jamwaffles/embedded-graphics/pull/351) Removed the `egcircle!`, `egellipse!`, `egrectangle!`, `egline!`, `egrectangle!`, `egroundedrectangle!` and `egtriangle!` macros.
```

Comment:
`egellipse!` and `egroundedrectangle!` were never included in a release and I think they shouldn't be included in the changelog.